### PR TITLE
Update google-common-protos

### PR DESCRIPTION
**Pull request by andreamlin**

### CODE REVIEW

**Comment by andreamlin** [comment]
openjdk7 has a failing build because of Java 7, but openjdk8 passes!

PTAL

File: generated/java/proto-google-common-protos/src/main/proto/google/logging/v2/logging_metrics.proto [hunk 1]
```diff
@@ -0,0 +1,263 @@
+// Copyright 2018 Google Inc.
```
Comment (garrettjonesgoogle) [inline generated/java/proto-google-common-protos/src/main/proto/google/logging/v2/logging_metrics.proto]: Logging service protos shouldn't be here

File: generated/java/proto-google-common-protos/src/main/proto/google/api/servicemanagement/v1/servicemanager.proto [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api.servicemanagement.v1;
```
Comment (garrettjonesgoogle) [inline generated/java/proto-google-common-protos/src/main/proto/google/api/servicemanagement/v1/servicemanager.proto]: Service management protos shouldn't be here

File: generated/java/proto-google-common-protos/src/main/proto/google/api/servicecontrol/v1/quota_controller.proto [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.api.servicecontrol.v1;
```
Comment (garrettjonesgoogle) [inline generated/java/proto-google-common-protos/src/main/proto/google/api/servicecontrol/v1/quota_controller.proto]: service control protos shouldn't be here

**Pull request merged**